### PR TITLE
Verbose mode for flow info

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- Adding `--verbose` to `cci flow info` will output all flow options defined.

# Issues Closed

### CODE REVIEW

File: cumulusci/core/flowrunner.py [hunk 1]
```diff
@@ -373,15 +376,33 @@ def get_flow_steps(self, for_docs=False):
...
                 new_source = ""
 
+            if step.task_config.get("options"):
+                options_defined = True
+                if verbose:
+                    options = step.task_config.get("options")
+                    options_info = f"{padding}  options:"
+
+                    for option, value in options.items():
+                        options_info += f"\n{padding}{padding} {option}: {value}"
```
Comment (jstvz) [inline cumulusci/core/flowrunner.py:386]: These lines aren't covered by a test.

File: cumulusci/core/flowrunner.py [hunk 2]
```diff
@@ -373,15 +376,33 @@ def get_flow_steps(self, for_docs=False):
...
             lines.append(
-                f"{'    ' * (i + 1)}{steps[i + 1]}) task: {task_name}{new_source}"
+                f"{'    ' * (i + 1)}{steps[i + 1]}) task: {task_name}{new_source}{has_options}"
             )
+
             if when:
                 lines.append(when)
 
+            if options_info:
+                lines.append(options_info)
```
Comment (jstvz) [inline cumulusci/core/flowrunner.py:398]: This branch is uncovered.

**Review by jstvz — CHANGES_REQUESTED** [review]
🚀

---
You've got some test failures to resolve here. You'll also need to add coverage for your conditionals by adding tests to test_flowrunner.py: https://github.com/SFDO-Tooling/CumulusCI/blob/d535ae6c51fccce7995be348bf66fd81f1562e37/cumulusci/core/tests/test_flowrunner.py#L146-L162

**Review by jstvz — CHANGES_REQUESTED** [review]
Sorry, I linked to the wrong example in my original review. I meant to reference this test that covers the `for_docs` parameter in test_flowrunner: https://github.com/SFDO-Tooling/CumulusCI/blob/d535ae6c51fccce7995be348bf66fd81f1562e37/cumulusci/core/tests/test_flowrunner.py#L180-L194

As a general rule, we test each module's behavior in its corresponding test module. We'd want to also verify that the option's behavior is correct there.

**Review by davisagli — COMMENTED** [review]
I tried it out, code and tests lgtm. One tiny suggestion.

**Comment by bethbrains** [comment]
[W-8887744](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000096YIXIA2/view)

**Review by Br4nd0R — APPROVED** [review]
This lgtm, nice work @bethbrains!

**Comment by davisagli** [comment]
@bethbrains please update the suggested release note in the PR description since we ended up not exposing the `--verbose` flag

**Comment by jstvz** [comment]
Closes #2835

**Pull request merged**